src/core/src/op/slice.cpp

```diff
@@ -117,7 +117,7 @@ bool Slice::has_evaluate() const {
 }
 
 bool Slice::can_constant_fold(const OutputVector& inputs) const {
-    return inputs.at(0).get_element_type().bitwidth() >= 8;
+    return inputs.at(0).get_element_type().bitwidth() >= 8 && Op::can_constant_fold(inputs);
```

Comment:
```suggestion
    return inputs.at(0).get_element_type().bitwidth() >= 8 && Node::can_constant_fold(inputs);
```
or Op have different version?

Reply:
No, `Op` doesn't override that. But `Slice` directly derives from `Op` and bypassing it to `Node::can_co...` makes another concern. IMO calling `Op::can_co...` is straight forward.